org.videolan.VLC.Plugin.fdkaac.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Should create an appdata file so it shows up as installable in stores.

Reply:
Indeed will do thanks.